tests/unit/test_zowe_core.py

```diff
@@ -68,6 +69,14 @@ def test_object_should_be_instance_of_class(self):
         request_handler = RequestHandler(self.session_arguments)
         self.assertIsInstance(request_handler, RequestHandler)
 
+    @mock.patch('requests.Session.send')
+    def test_perform_streamed_request(self, mock_send_request):
+        """Performing a streamed request should call 'send_request' method"""
+        mock_send_request.return_value = mock.Mock(status_code=200)
+        request_handler = RequestHandler(self.session_arguments)
+        request_handler.perform_streamed_request("GET", {'url': 'https://www.zowe.org'})
+        mock_send_request.assert_called_once()
```

Comment:
Thanks for adding the unit test, now the patch coverage for this PR is >80% 🎉 

In the unit tests, we want to validate that the important behavior of methods works as expected. In this case, the `perform_streamed_request` method needs to pass `stream=True` when it sends a request.

To test this, I would suggest adding another assertion at the end of the test:
`self.assertTrue(mock_send_request.call_args.kwargs["stream"])`
And if the assertion fails, then please fix the behavior of the `perform_streamed_request` method so it passes 🙂

Reply:
Oops, I didn't realize that `call_args.kwargs` only exists in Python 3.8 and newer.

Replacing it with `call_args[1]` should work in older versions of Python, like this:
`self.assertTrue(mock_send_request.call_args[1]["stream"])`